### DISCUSSION TITLE
Erro ao cadastrar chave pix

### DISCUSSION THREAD

**Discussion by NagatoInformatica** [RamonSilva20/mapos#2289] [General]
Olá
Quando tento cadastrar a chave pix em Configurações / Sistema / Financeiro aparece uma tela com esse erro da foto. Tentei reinstalar o sistema e o problema ainda persiste .
Alguém pode me ajudar?
Obrigado
![Captura de tela 2023-11-30 081327](https://github.com/RamonSilva20/mapos/assets/152493185/849d76d0-1ef7-47bf-b1c3-47f951118ad5)

**Comment by Pr3d4dor** [comment]
Está faltando as bibliotecas do composer, precisa rodar: composer install

**Reply by NagatoInformatica** [reply]
Primeiramente agradeço a atenção.
Meu sistema está hospedado no Hostinger e conferir que a instalação do Composer está ativa. Só não sei conferir se precisa de algo mais nessa instalação. 
No de mais, o sistema está funcionando perfeitamente.

**Reply by Pr3d4dor** [reply]
Você precisa rodar novamente o comando composer install ou composer update
